static/app/components/commitLink.tsx

```diff
@@ -1,94 +1,88 @@
-import {Button} from 'sentry/components/button';
-import ExternalLink from 'sentry/components/links/externalLink';
-import {IconBitbucket, IconGithub, IconGitlab, IconVsts} from 'sentry/icons';
-import {t} from 'sentry/locale';
-import {Repository} from 'sentry/types';
-import {getShortCommitHash} from 'sentry/utils';
+import { Button } from "sentry/components/button";
+import ExternalLink from "sentry/components/links/externalLink";
+import { IconBitbucket, IconGitHub, IconGitlab, IconVsts } from "sentry/icons";
+import { t } from "sentry/locale";
+import { Repository } from "sentry/types";
+import { getShortCommitHash } from "sentry/utils";
 
 type CommitFormatterParameters = {
-  baseUrl: string;
-  commitId: string;
+    baseUrl: string;
+    commitId: string;
 };
 
 type CommitProvider = {
-  commitUrl: (opts: CommitFormatterParameters) => string;
-  icon: React.ReactNode;
-  providerIds: string[];
+    commitUrl: (opts: CommitFormatterParameters) => string;
+    icon: React.ReactNode;
+    providerIds: string[];
```

Comment:
None of this is related `Github/GitHub`. 

We have a convention and this breaks our code-styles.